### ISSUE TITLE
Configuration screen has less features after update

### ISSUE THREAD

**Issue by AllanKley** [GeekMagicClock/smalltv#12]
I was having some problems with the weather so I decided to update the firmware to the latest version. That ended up causing more problems so I decide to go for the V3.0.33 to see if it would work and it did. The only problem is that the configuration screen before all the updates had more features (I could for example go to 192.168.18.211/images and configure my images to cycle between them), now I can only select a one from a couple of gifs and they are not even appering

**Comment by GeekMagicClock** [comment]
Hi, recently the default public weather api key is run out, just need to update the key, you could register for your own, it's free.
https://home.openweathermap.org/api_keys
And you have to update the correct firmware for that model you have.

**Comment by AllanKley** [comment]
I have already registered a new api key, the weather is working properly and even the little gif on the bottom right works if I upload it myself. The only thing is that before I put it on v3.0.33 I was able to switch modes, for example forecast, different clock faces and even just a jpg but now on v3.0.33 I can only have it on the one in the image. Is there a way for me to see which firmwares are correct for my device?
![20231214_223542](https://github.com/GeekMagicClock/smalltv/assets/90280541/24b568cf-78f3-4395-ad74-d5ec0896a03c)
![Screenshot_20231214_223641_Chrome](https://github.com/GeekMagicClock/smalltv/assets/90280541/f5616d26-abbb-4e3b-b631-318bb66bffee)
